index.html

```diff
@@ -737,7 +740,8 @@ <h4>
         <a data-cite="webdriver2/#dfn-current-browsing-context">current browsing
         context</a>'s [=top-level traversable=]'s [=navigable/active document=].
         </li>
-        <li>Set |document|.{{Document/[[PostureOverride]]}} to |posture|.
+        <li>Set |document|.<a data-link-for="top-level traversable">
```

Comment:
This is still wrong.

Reply:
https://github.com/JuhaVainio/W3C-device-posture/commit/9192ea0141132ca88512739cf0d0d7db8e39dd40